src/pages/console/authz/permissions.mdx

```diff
@@ -16,26 +16,26 @@ You are going to define set up role-based authorization rules for each of the mo
 2. In the **Model** pane on the right, expand the **Owners** window. 
 3. Choose **Create**, **Read**, **Update** and **Delete** to specify that *Owners* have create, read, update, and delete access. The settings look as follows.
 
-![Owner authorization rules on a GraphQL data model](/images/console/10_ownersaccess.png)
+![](/images/console/10_ownersaccess.png)
 
 ## To set a private authorization rule
 1. Using the *Books* data model that you created in the [Create a data model example](/console/data/data-model#Create-a-data-model-example), set the authorization mode to **Cognito user pool**. 
 2. In the **Model** pane on the right, expand the **Any signed-in users** window. 
 3. Choose **Create**, **Read**, and **Update** to specify that any signed-in authenticated user has create, read, and update, access. The settings look as follows.
 
-![Private authorization rules on a GraphQL data model](/images/console/11_privatesaccess.png)
+![](/images/console/11_privatesaccess.png)
```

Comment:
Since we're making the image decorative, I think we shouldn't have the "The settings look as follows" sentence above it. 

If it's decorative then it means it can be removed and the content wouldn't be missing anything. So I think it makes sense to not mention it at all.

Hopefully that makes sense